sn_node/src/put_validation.rs

```diff
@@ -12,7 +12,7 @@ use sn_networking::{get_raw_signed_spends_from_record, GetRecordError, NetworkEr
 use sn_protocol::{
     storage::{
         try_deserialize_record, try_serialize_record, Chunk, RecordHeader, RecordKind, RecordType,
-        SpendAddress,
+        Scratchpad, SpendAddress,
```

Comment:
I am not sure of this name, it feels too loose. Perhaps Account Packet is more clear?

Reply:
I think it's worth there being a separation between the data type on the network and the data being stored.

There's nothing that says this _must_ be an account packet. There could be other usecases for this eg.

So I've been leaning into `AccountPacket` being the data structure + representation of a user's `Account` space (or whatever it ends up being called).

And with that I feel this data type needs a more generic name. 


`Scratchpad` i agree doesn't feel the best, but was the best I could come up with for that. Happy to change to something else to better describe this.

Reply:
After some chat, we agreed on keeping `Scratchpad` as it is reflective of the usecase